module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -597,15 +600,24 @@ namespace platform {
                     // Htw.translation() = (o.rows(MotionModel::PX, MotionModel::PZ));
                     sensors->Htw = convert<double, 4, 4>(Htw);
 
+                    // Integrate gyro to get angular positions
+                    theta += o.rows(MotionModel::WX, MotionModel::WZ) * 1.0 / 90.0;
```

Comment:
Remove the gyroscope integration and instead use the filter. If it's not working it should be fixed